### PR TITLE
Call /v1/twitter/create occurs name too long error#25

### PR DESCRIPTION
**Pull request by minhtuan412**
Finish call twitter api to upload image
capture

### CODE REVIEW

**Comment by hitoshi5858** [comment]
@minhtuan412  Please link issue. You can see it.
https://www.github.wiki/en/github-ae@latest/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: api/.env_sample [hunk 1]
```diff
@@ -2,8 +2,9 @@
 USE_DYNAMODB_LOCAL=true
 DYNAMODB_LOCAL_URL=http://localhost:8000
 
-CONSUMER_KEY=xGwoSRAdx8FQoc2WKYVGUsFtJ
-CONSUMER_SECRET=Ly0Bl8om7HGPToPjNoPp8KAlzhpmQiUTFJtmB9HW0tCAXWXhIv
-BEARER_TOKEN=    "AAAAAAAAAAAAAAAAAAAAAD6zewEAAAAAY2m5dOB9nJGp3I%2FXqOZYvAqd%2Fpw%3DYwKOHI3nLvXxdxNtD0OZgw1bWGGanqvzgIw5pQ96iGOgylsh8z",
-
+CONSUMER_KEY=w8PP8LhATEHiUXv7oW5TeCA4L
+CONSUMER_SECRET=mTZ0xCEO3Jp1WHH8QgjjIlicShrv4o1Rsj73CKKIsXaMTPVdvI
+BEARER_TOKEN=AAAAAAAAAAAAAAAAAAAAAJ7KfQEAAAAA7C8AaPjXbfgD0s5miZcRnkixA70%3D9y0sDRIccQkKW6EKTzELbvxX7FDuF0iGx0E7JorokZQkjF1rPm
```
Comment (hanchenglin) [inline api/.env_sample:7]: @minhtuan412  Please delete items（BEARER_TOKEN） you don't need

File: api/.env_sample [hunk 2]
```diff
@@ -2,8 +2,9 @@
...
+ACCESS_TOKEN_KEY=1552233873949736963-oC4qjo2zW8oJKrwX3urzTif0m0OJeR
+ACCESS_TOKEN_SECRET=A08peEcVqftsGvTxvvy4s1iHm1fl75wdEEqtubWueUXOi
```
Comment (hanchenglin) [inline api/.env_sample:9]: @minhtuan412  Please set value as blank, do not set personal account information

File: api/.env_sample (hunk 1 above)
Comment (minhtuan412) [reply api/.env_sample]: I deleted it

File: api/.env_sample (hunk 2 above)
Comment (minhtuan412) [reply api/.env_sample]: I updated it

File: api/.env_sample [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
-CONSUMER_KEY=w8PP8LhATEHiUXv7oW5TeCA4L
-CONSUMER_SECRET=mTZ0xCEO3Jp1WHH8QgjjIlicShrv4o1Rsj73CKKIsXaMTPVdvI
-BEARER_TOKEN=AAAAAAAAAAAAAAAAAAAAAJ7KfQEAAAAA7C8AaPjXbfgD0s5miZcRnkixA70%3D9y0sDRIccQkKW6EKTzELbvxX7FDuF0iGx0E7JorokZQkjF1rPm
-ACCESS_TOKEN_KEY=1552233873949736963-oC4qjo2zW8oJKrwX3urzTif0m0OJeR
-ACCESS_TOKEN_SECRET=A08peEcVqftsGvTxvvy4s1iHm1fl75wdEEqtubWueUXOi
+## connect twitter developer to get key
+# CONSUMER_KEY=
+# CONSUMER_SECRET=
+# ACCESS_TOKEN_KEY=
+# ACCESS_TOKEN_SECRET=
```
Comment (hanchenglin) [inline api/.env_sample:9]:
```suggestion
CONSUMER_KEY=
CONSUMER_SECRET=
ACCESS_TOKEN_KEY=
ACCESS_TOKEN_SECRET=
```

**Pull request merged**